SPEC.md

```diff
@@ -7788,7 +7816,7 @@ Example input:
 
 ```json
 {
-  "fastq": "sample1234_R1.fastq"
+  "test_matches.fastq": "sample1234_R1.fastq"
```

Comment:
Testing match alone is probably not enough to add a new `tests/data` file. Suggest removing those generated FASTQ files now and modifying the test to use one of the files.

Reply:
Changing from fastq to person.json